### DISCUSSION TITLE
What do you think about these baklava terms

### DISCUSSION THREAD

**Discussion by 0o001** [Trendyol/baklava#897] [Ideas]
Hello everyone,

While thinking about the Baklava Design System, I thought about these three terms.
If you like, maybe we can create an etymology like this.
I think it can also be useful for features that are being developed in the future.

It provides a structured approach to UI development by categorizing components into three main levels: Slices, Servings, and Trays.

**Slice**

Slices are the primitive components in Baklava Design System & Baklava Slice. They are the fundamental building blocks for more complex UI elements. Examples include buttons, input fields, and icons. These components are designed to be flexible and reusable across different parts of an application.

**Serving**

Servings are composite UI parts made by combining multiple Slices. They represent common UI patterns and frequently used components, such as nav bars, chat messages, and footers. Servings help speed up development by providing pre-built, customizable UI elements that ensure a consistent look and feel.

**Trays**

Trays are complete pages or sections of a web application, created by assembling multiple Servings. They represent fully functional and structured parts of an application, such as an admin dashboard, a blog, or a user profile page. Trays offer a cohesive view of how different Servings work together to form a comprehensive user interface.

**Comment by miratcan** [comment]
Slice has destructive meaning, it's a thing that when you corrupt or broke something. That does not contain meaning of construction.
